components/Conversations/Detail/ConversationDetail.tsx

```diff
@@ -72,16 +66,9 @@ const useTranscriptionTimer = (
 const ConversationDetail: React.FC<ConversationDetailProps> = ({ conversation }) => {
   const { micActivity, isAudioOn, saveCurrentConversation } = useConversations()
   const isTranscribing = useTranscriptionTimer(isAudioOn, micActivity, saveCurrentConversation)
-  const [savedConversation, setSavedConversation] = useState<ConversationData | null>(conversation || null)
-
- useEffect(() => {
-    if (conversation) {
-      setSavedConversation(conversation)
-    }
-  }, [conversation])
 
-  if (savedConversation) {
-    return <CompletedConversation conversation={savedConversation} />
+  if (conversation) {
+    return <CompletedConversation conversation={conversation} />
   }
```

Comment:
@Jungyoonlim it's best to only have one variable that controls what conversation is selected. right now that's the `conversation` props that's passed in (which doesn't actually need to be a prop because we can just get it from userConversations` directly

```
  const { selectedConversationId, conversations } = useConversations()
  const selectedConversation = conversations.find((conv) => conv.id === selectedConversationId)
```

Reply:
When `selectedConversationId` is null, active transcript should be shown. Having to track `savedConversation` in addition to `selectedConversationId` will just add extra complexity/confusion. Here, since `savedConversation` was never cleared, `ActiveConversation` was never accessible